doc/index.rst

```diff
@@ -4,41 +4,113 @@
+----------------------------------
 
 Matplotlib is a comprehensive library for creating static, animated,
 and interactive visualizations in Python.
 
-Learn
-=====
-      
-- :doc:`Quick-start Guide <tutorials/introductory/usage>`
-- Basic :doc:`Plot Types <plot_types/index>` and :doc:`Example Gallery <gallery/index>`
-- `Introductory Tutorials <../tutorials/index.html#introductory>`_
-- :doc:`External Learning Resources <resources/index>`
-      
-Reference
-=========
+Installation
+============
 
-- :doc:`API Reference <api/index>`
+.. panels::
+    :card: + install-card
+    :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
 
-  - :doc:`pyplot API <api/pyplot_summary>`: top-level interface to create 
-    Figures (`.pyplot.figure`) and Subplots (`.pyplot.subplots`, 
-    `.pyplot.subplot_mosaic`)
-  - :doc:`Axes API <api/axes_api>` for *most* plotting methods
-  - :doc:`Figure API <api/figure_api>` for figure-level methods
+    Installing using conda
+    ^^^^^^^^^^^^^^^^^^^
 
-How-tos
-=======
+    Matplotlib is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__
+    distribution and can be installed with Anaconda or Miniconda:
 
-- :doc:`Installation Guide <users/installing>`
-- :doc:`Contributing to Matplotlib <devel/index>`
-- :doc:`Matplotlib FAQ <faq/index>`
 
-Understand how Matplotlib works
-===============================
+    .. code-block:: bash
 
-- Many of the :doc:`Tutorials <tutorials/index>` have explanatory material
+        conda install matplotlib
+
+    ---
+
+    Installing using pip
+    ^^^^^^^^^^^
+
+    Matplotlib provides wheels for Linux, OSX, and Windows which can be
+    installed via pip from `PyPI <https://pypi.org/project/matplotlib>`__.
+
+
+    .. code-block:: bash
+
+        pip install matplotlib
+
+
+Further details are availabe in the :doc:`Installation Guide <users/installing>`.
+
+
+Learning resources
+==================
+
+
+.. panels::
+
+    Tutorials
+    ^^^^^^^^^
+
+    - :doc:`Quick-start Guide <tutorials/introductory/usage>`
+    - Basic :doc:`Plot Types <plot_types/index>`
+    - `Introductory Tutorials <../tutorials/index.html#introductory>`_
+    - :doc:`External Learning Resources <resources/index>`
+
+    ---
+
+    How-tos
+    ^^^^^^^
+    - :doc:`Example Gallery <gallery/index>`
+    - :doc:`Matplotlib FAQ <faq/index>`
+
+    ---
+
+    Understand how Matplotlib works
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    - The :ref:`users-guide-explain` section of the :doc:`Users guide <users/index>`
+    - Many of the :doc:`Tutorials <tutorials/index>` have explanatory material
+
+    ---
+
+    Reference
+    ^^^^^^^^^
+
+    - :doc:`API Reference <api/index>`
+
+      - :doc:`Axes API <api/axes_api>` for *most* plotting methods
+      - :doc:`Figure API <api/figure_api>` for figure-level methods
+      - Top-level interface to create Figures (`.pyplot.figure`) and Subplots
+        (`.pyplot.subplots`, `.pyplot.subplot_mosaic`)
+
+    - :doc:`Extra Toolkits <api/toolkits/index>`
```

Comment:
Remove this bullet point because toolkits are part of the API index now (#21003).